staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/README.md

```diff
@@ -16,7 +16,7 @@ This plugin provides an integration with Azure Active Directory device flow. If
    ```
    --oidc-client-id="spn:APISERVER_APPLICATION_ID" \
    --oidc-issuer-url="https://sts.windows.net/TENANT_ID/"
-   --oidc-username-claim="sub"
+   --oidc-username-claim="upn"
    ```
```

Comment:
@dwatrous is the email address expected in this claim? The AAD stores the user unique ID in the sub claim https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-token-and-claims.

Reply:
I'm not sure if it's configurable, but in my organization, `upn` is the user's email address. the `sub` claim is always a hash/guid of some sort `"sub":"92d0312b-26b9-4887-a338-7b00fb3c5eab"`. I think the reason is that a user may have multiple representations in AAD with the same email address, but they correspond to different records.